openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1729,7 +1729,7 @@ The following additional Credential Issuer metadata are defined for this Credent
 * `credential_definition`: REQUIRED. Object containing the detailed description of the Credential type. It consists at least of the following two sub claims:
   * `type`: REQUIRED. Array designating the types a certain Credential type supports according to [@VC_DATA], Section 4.3.
   * `credentialSubject`: OPTIONAL. An object containing a list of name/value pairs, where each name identifies a claim offered in the Credential. The value can be another such object (nested data structures), or an array of such objects. To express the specifics about the claim, the most deeply nested value MAY be an object that includes a following non-exhaustive list of parameters defined by this specification:
-      * `mandatory`: OPTIONAL. Boolean which when set to `true` indicates the claim MUST be present in the issued Credential. If the `mandatory` property is omitted its default should be assumed to be `false`.
+      * `mandatory`: OPTIONAL. Boolean which, when set to `true`, indicates that the Credential Issuer will always include this claim in the issued Credential. If the `mandatory` property is omitted its default should be assumed to be `false`.
```

Comment:
what about `mandatory` -> `required`?

Reply:
I find the parameter `required` used in other context and more recognizable than `mandatory`

Reply:
I don't have a strong opinion here.. suggest we bikeshed in the WG call/this PR comment.

Reply:
I think mandatory is better language because its the issuer saying the claim is mandatory in this credential they issue. "required" would kind of be a requirement the issuer is placing on its self in this context which makes less sense to me.

Reply:
Discussed on today's editors and no one felt required was significantly better, and I kind of agree with Tobias that mandatory is slightly better for this case.

Reply:
I can live with either, and I think 'mandatory' is just fine.

Reply:
@peppelinux I think we are sticking with `mandatory`